### ISSUE TITLE
Items won't actually automatically be moved from inbox when there is space in inventory

### ISSUE THREAD

**Issue by IntrepidCurmudgeon** [DexterHuang/CyberCodeOnline#2304]
The Item Inbox currently says "Items here will be automatically moved to your inventory once you have enough space", but that is not the case. Most players don't want that feature, anyways. We want to be able to pick and choose which items to take from the Item Inbox (similar to how looting enemies works).

**Comment by CyberMechani** [comment]
Yeah, that's a good idea. Plus I agree, I want to pick what items I take and what I can save for later.
